numpy/core/fromnumeric.py

```diff
@@ -1159,10 +1159,10 @@ def argmin(a, axis=None, out=None):
     >>> a[ind]
     0
```

Comment:
This line is out of date.

Reply:
Oh okay, they were outdated in the current master then?

Reply:
should I remove them here or create another branch for that purpose?